transformations/americanize_britishize_english/transformation.py

```diff
@@ -0,0 +1,132 @@
+import requests
+
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+
+
+def britishize_americanize(string, custom_vocab):
+    """
+    Gets the british to american english dictionary
+    Gets the american to British english dictionary
+    Merges both these dictionaries with the custom vocab dictionary.
+    Parameters:
+        string(str): original string
+        custom_vocab(dict): custom vocab dictionary containing words that are not present in the imported dictionaries
+    Returns:
+        str: String after replacing the words
+    """
+
+    url = "https://raw.githubusercontent.com/hyperreality/American-British-English-Translator/master/data/american_spellings.json"
```

Comment:
Both the requests.url callings can be done in the constructor itself. Would be unnecessary to repeat it for every incoming sentence call.

Reply:
Makes absolute sense. Changes done. Kindly review.